build.gradle

```diff
@@ -52,3 +53,7 @@ android {
         abortOnError false
     }
 }
+
+dependencies {
+    implementation 'com.android.support:support-annotations:27.0.2'
```

Comment:
Why do we need this? I'm not a big fan of the support library and try to avoid it.

Reply:
My guess is that it's just annotations part of it for making code look cleaner (e.g. `@NonNull` or `@Nullable` etc. JSR-305)

Make it "compileOnly" so It doesn't add anything to runtime.

Reply:
Yes, that is my intention for supporting the annotations.
I did not know about "compileOnly", will do. I agree it will be nice to make sure there is no extra unneeded runtime dependency.

Reply:
Welcome aboard, I see you're catching up pretty quickly ;)